src/utils/operator.ts

```diff
@@ -173,14 +176,16 @@ export function getOrderSurplus(order: RawOrder): Surplus {
  * @param order The order
  */
 export function getOrderExecutedAmounts(
-  order: Pick<RawOrder, 'executedBuyAmount' | 'executedSellAmount' | 'executedFeeAmount'>,
+  order: Pick<RawOrder, 'executedBuyAmount' | 'executedSellAmount' | 'executedFeeAmount' | 'executedSurplusFee'>,
 ): {
   executedBuyAmount: BigNumber
   executedSellAmount: BigNumber
 } {
+  const { executedBuyAmount, executedSellAmount, executedFeeAmount, executedSurplusFee } = order
+  const totalFee = new BigNumber(executedFeeAmount).plus(executedSurplusFee)
```

Comment:
Considering https://github.com/cowprotocol/cowswap/pull/1588#discussion_r1037501631 it should be:

```typescript
const totalFee = new BigNumber(executedSurplusFee ?? executedFeeAmount)
```

Reply:
There's an open discussion with this, yes,I will update with the resolution of it